Doc: second draft of 8.1 what's new (#37370)
grafana/grafana: docs/sources/whatsnew/whats-new-in-v8-1.md

> **Note:** This topic will be updated frequently between now and the final release.

> Grafana 8.1 builds upon our promise of composable, open observability platform with new panels and extends functionality launched in Grafana 8.0. We’ve got new panels, including the Geomap panel, the Annotations panel, and some great updates to the Time Series panel. For enterprise customers we’ve got additions to fine grained access control, updates to the reporting scheduler and more. We’ve also got some new transformations, updates to data sources and more. Details below.

Reviewer: ```suggestion
Grafana 8.1 builds upon our promise of a composable, open observability platform with new panels and extends functionality launched in Grafana 8.0. We’ve got new panels, including the Geomap panel, the Annotations panel, and some great updates to the Time Series panel that was introduced in the 8.0 release. We’ve also got some new transformations, and updates to data sources. We’ve got additions to fine grained access control, updates to the reporting scheduler, and more for our enterprise customers. Details below.
```

> We’ve summarized what’s new in the release here, but you might also be interested in the announcement blog post as well. If you’d like all the details you can checkout the complete [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).

Reviewer: ```suggestion
In addition to what we have summarized here, you might also be interested in the announcement blog post. For all the details, check out the complete [CHANGELOG.md](https://github.com/grafana/grafana/blob/master/CHANGELOG.md).
```

Reviewer: I am really happy to see this introduction! It sets up the readers expectations nicely

Author: It was all your idea - thank you!

## Grafana OSS features

These features are included in the Grafana open source edition.

> ### Time series panel updates

Reviewer: We should move this down to after Geomap and Annonations panel sections.

Author: Good point, are you willing to do that?

Reviewer: Sure
